Mage/src/main/java/mage/abilities/common/BoastAbility.java

```diff
@@ -0,0 +1,82 @@
+package mage.abilities.common;
+
+import mage.MageObjectReference;
+import mage.abilities.Ability;
+import mage.abilities.ActivatedAbilityImpl;
+import mage.abilities.condition.Condition;
+import mage.abilities.costs.Cost;
+import mage.abilities.costs.Costs;
+import mage.abilities.costs.mana.ManaCosts;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.Effects;
+import mage.constants.EffectType;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.watchers.common.AttackedThisTurnWatcher;
+
+/**
+ *
+ * @author weirddan455
+ */
+public class BoastAbility extends ActivatedAbilityImpl {
+
+    private static final Effects emptyEffects = new Effects();
+
+    public BoastAbility(Effect effect, ManaCosts cost) {
```

Comment:
You probably don't need to do most of this stuff that's copied from the `ConditionalActivatedAbility` class, if we need more constructors we can add them later.